Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
         booleanParam(name: 'ppc64le', defaultValue: true, description: 'PowerPC (ppc64le) Build/Test')
         booleanParam(name: 'windowsRS1', defaultValue: false, description: 'Windows 2016 (RS1) Build/Test')
         booleanParam(name: 'windowsRS5', defaultValue: true, description: 'Windows 2019 (RS5) Build/Test')
-        booleanParam(name: 'dco', defaultValue: true, description: 'Run the DCO check')
+        booleanParam(name: 'dco', defaultValue: false, description: 'Run the DCO check')
```

Comment:
Unfortunately Jenkins uses Jenkinsfile from master branch so this does not help...

Reply:
@cpuguy83 you can use "build with parameters" instead

Reply:
I don't see a way to. Funny thing is last time it totally built and didn't run the dco check.

Reply:
Yeah, Jenkins often caches parameters; sometimes it keeps the old ones; it's in the "classic" view, then click the PR number, and then it shows in the left-hand menu 
![8461F156-B950-4B02-8106-6C54F2A643C8](https://user-images.githubusercontent.com/1804568/114248096-1b2d5800-9997-11eb-9aa3-0b473c349799.jpeg)
![CB351E73-B2AE-4519-9499-4755BD2E524B](https://user-images.githubusercontent.com/1804568/114248104-1cf71b80-9997-11eb-9c4f-0b33de21e4d3.jpeg)

Reply:
Oh right! Classic view.